adam/situation/templates/phase1_templates.py

```diff
@@ -444,6 +459,12 @@ def _instantiate_situation(
             ],
             syntax_hints=template.syntax_hints,
             axis_info=self._compute_axis_info(object_var_to_instantiations),
+            gazed_objects=immutableset(
```

Comment:
Why do we allow `Region`s to be gazed at?

Reply:
@gabbard By default, we grab the `Defaults to all semantic role fillers of template actions.` and regions can fill a role such as a `GOAL` in an action.

Reply:
@LichMaster98 : I am confused by:
> By default, we grab the Defaults to all semantic role fillers of template actions

Can you clarify?

Reply:
@gabbard Yes. The image below is an instance of the "put" curriculum in our phase 1 curriculum. Where the semantic role fillers for the action are `dog`, `hat`, and `Region(reference_object=car, distance=exterior-but-in-contact, direction=+_GravitationalAxis())`

The default which is used in the scene generation code grabs all three of these and adds them to a list of gazed items. I copied this implementation into the template builder to allow for specifying explicit items which are being gazed at in the template scene. Because regions can appear as semantic role fillers (as in this example) I handled them in a way that would mirror how the situation code appears to be handling them.

![image](https://user-images.githubusercontent.com/8626833/70452579-5a7f2800-1a75-11ea-8a52-3b87d9f23da5.png)

Reply:
I think this is a bug in the situation code.  Can you make an issue to block `Region`s from being gazeable?

Reply:
And I'll fix the template code to block regions as well.